server/vulnerabilities/vuln_ubuntu/ubuntu.go

```diff
@@ -0,0 +1,491 @@
+// Package vuln_ubuntu contains a ParseUbuntuRepository method to parse the Ubuntu repository
+// to look out for Ubuntu releases that patch CVEs. It parses the changelogs from the metadata.
+//
+// It also contains a LoadUbuntuFixedCVEs to load the results of the parsing.
+//
+// Both the parsing and loading of results use sqlite3 as backend storage.
+package vuln_ubuntu
+
+import (
+	"archive/tar"
+	"bufio"
+	"compress/bzip2"
+	"compress/gzip"
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/gocolly/colly"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/ulikunitz/xz"
+	"golang.org/x/sync/errgroup"
+)
+
+func defaultCacheDir() (string, error) {
+	return filepath.Join("vuln", "ubuntu"), nil
+}
+
+// UbuntuPkg holds data to identify a Ubuntu package.
```

Comment:
Let's add a comment explaining that there's currently no clear way to define `arch` in the parsing.